### PR TITLE
[FEAT] [TESTNET] Change testnet chain name from Mumbai to Amoy

### PR DESCRIPTION
**Pull request by eliasSFL**
# Description

~~- Updated .env.sample with the amoy contracts~~
- Updated Block Explorer URL to Amoy Polygonscan
- Change network from Mumbai to Amoy

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,62 +1,60 @@
-# VITE_API_URL=<API_URL>
+# VITE_API_URL=https://api-dev.sunflower-land.com
 
 # VITE_NETWORK=mainnet
-VITE_NETWORK=mumbai
+VITE_NETWORK=amoy
```
Comment (spencerdezartsmith) [inline .env.sample:4]: Not sure what happened here but this line is the only one that should change. 

It looks like maybe you have committed your own `.env` as these are old contract values. 🤷🏼

**Review by spencerdezartsmith — CHANGES_REQUESTED** [review]
Nice one. Looking good except for the `env.sample` commit. This should only have the one line change in it.

File: .env.sample (hunk 1 above)
Comment (eliasSFL) [reply .env.sample:4]: @spencerdezartsmith I took these contracts from testnet.yml though. I noticed that the .env.sample wasn't changed when the migration happened so I updated it with the new contracts. maybe you want to cross check with that file to make sure the contracts match?

Comment (eliasSFL) [reply .env.sample:4]: image
As you can see here the `testnet.yml` contains the new Amoy contracts, but `.env.sample` still has the Mumbai contracts. I merely just copied the Amoy Contracts from `testnet.yml` over.

**Review by spencerdezartsmith — APPROVED** [review]
LGTM

**Pull request merged**